atm.js

```diff
@@ -1,3 +1,101 @@
+// var depositInt = parseInt (0)
+var savingsBal = parseInt (0)
+var checkingBal = parseInt (0)
+var totalBal = parseInt (0)
+
+function changeClass(checkingBal){
+   $("#checking .balance").removeClass("zero");
+   if (checkingBal === 0) {
+    $("#checking .balance").addClass("zero");
+   }
+};
+
+function changeSClass(savingsBal){
+   $("#savings .balance").removeClass("zero");
+   if (savingsBal === 0) {
+    $("#savings .balance").addClass("zero");
+   }
+};
```

Comment:
These two functions are almost identical. Here's an idea for condensing them into one:

```js
var balance = {
    checking: 0,
    savings: 0
}

function changeClass(acctName){
    $("#" + acctName + " .balance").removeClass("zero");
    if (balance[acctName] === 0) {
        $("#" + acctName + " .balance").addClass("zero");
    }
};

changeClass("checking");
changeClass("savings");
```